### PR TITLE
Sorry I forgot to a pull request for Module_3

### PR DESCRIPTION
**Pull request by emma-luk**
Sorry I forgot to a pull request for Module_3

### CODE REVIEW

**Comment by jamdon** [comment]
You didn't implement any of the functions in the trucks.js module. Your server basically doesn't work - it crashes when I try to load the page because it tries to call a function that was never defined or implemented.

**Comment by emma-luk** [comment]
I have implemented the trucks module in Module_4 - I copied it into my trucks module for Module_3. Updated trucks.js

File: server.js [hunk 1]
```diff
@@ -11,4 +11,48 @@
...
+    //the output
+    var responseOutPut = 'Today is ' + date.getDayName() + ', '
+          + date.getDate('formatted') + '.' + '\n' + 'The food trucks available are:\n\n';
+
+    //trucks are open
+    var truckArray = trucks.filterByDay(date.getDayName());
+
+    //loop adding trucks
+    for (var i = 0; i < truckArray.length; i++) {
+        responseOutPut += '-' + truckArray[i].name + '\n';
```
Comment (jamdon) [inline server.js:44]: Your truckArray is just an array of names, not an array of objects with name properties. By trying to access a property that doesn't exist, it results in your code outputting undefined for each of the trucks.

**Comment by jamdon** [comment]
There was one issue with how you were trying to output the name of the truck (see comment inline). 👍

**Pull request closed without merging**